app/views/schools/teachers.html.erb

```diff
@@ -2,6 +2,16 @@
 
 <h3><strong>All Teachers</strong></h3>
 
+<%= form_tag("/schools/#{@school.id}/teachers", method: "get") do %>
+  <%= label_tag(:amount, "Filter by Salary - Greater Than:") %>
+  <%= text_field_tag(:amount) %>
+  <%= submit_tag("Find Teachers") %>
+<% end %>
+
+<p><%= link_to "Sort Teachers by Name", "/schools/#{@school.id}/teachers?sort=by_name" %></p>
```

Comment:
we dont need it to say param1 after the ? thats awesome!

Reply:
@duguy8 yeah! I just changed to param name to be sort instead of param1!